CHANGELOG.md

```diff
@@ -8,10 +8,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* Added validation of the master presence in replicaset and the 
+  master connection to the `utils.get_space` method before 
+  receiving the space from the connection (#331).
 
 ### Changed
 
 ### Fixed
+* Fixed a bug of the fiber status call when
```

Comment:
Let's make it simpler. Something like "fix fiber cancel on schema reload timeout"

Reply:
Simplified `CHANGELOG.md` and commit msg